src/index.js

```diff
@@ -1,3 +1,5 @@
 // @flow
-export {withFontLoading} from './with-font-loading';
-export {FontPlugin} from './plugin';
+import withFontLoading from './with-font-loading';
```

Comment:
linter prefers this pedantic style?

Reply:
Boils down to this basically: https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/prefer-default-export.md

I suppose we can change this to `export {default as withFontLoading} from ...`? I'll try that as well.